### DISCUSSION TITLE
Generate a "simulator landscape" page

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#3735] [Ideas]
The CNCF has a "landscape" that highlights different core infrastructures in various domains of cloud development/deployment. 
It provides an overview of the entire ecosystem of cloud tech.

https://landscape.cncf.io/

This would be really cool to do for simulators/systems bio tools. The curation status would be analog to the CNCF adoption categories, and different frameworks/tasks the categories.

**Comment by jonrkarr** [comment]
Similarly, it would be nice to have a page which summarizes the available projects. This what I had in mind with #3054. This could information information such as
- Taxonomic distribution of projects
- Distribution of projects by modeling frameworks, model languages
- Distribution of projects by sources
- Distribution of projects by year of creation
